tests/conftest.py

```diff
@@ -65,7 +66,7 @@ def skipif(items, whole_module=False):
             break
         # Skip if it won't run with IntelCompiler
         if i == 'cpu64-icc' and \
-           isinstance(configuration['compiler'], IntelCompiler) and \
+           isinstance(configuration['compiler'], (IntelCompiler, OneapiCompiler)) and \
```

Comment:
This wouldn't be needed if OneapiCompiler were a subclass of IntelCompiler as I think Mathias pointed out at some point, but yeah, nitpicking